src/javascript/app_2/App/Components/Form/input_field.jsx

```diff
@@ -43,8 +43,8 @@ const InputField = ({
 
     const changeValue = (e) => {
         if (type === 'number') {
-            const is_empty = !e.target.value || e.target.value === '';
-            const signed_regex = is_signed ? '^[\+\-]?[0-9]*(\.?([0-9]*)?)?$' : '^';
+            const is_empty = !e.target.value || e.target.value === '' || e.target.value === '  ';
+            const signed_regex = is_signed ? '[\+\-\.0-9]$' : '^';
```

Comment:
For `barrier` we need to check if value is leading with `+` or `-`. If you remove this it will falsely pass the validation in place for the `input_field` regex and return out of range error from API.